tiledb/common/algorithm/test/unit_parallel_merge.cc

```diff
@@ -30,6 +30,7 @@
 
 #include "tiledb/common/algorithm/parallel_merge.h"
 
+#include <test/support/assert_helpers.h>
```

Comment:
The changes to this file are not directly related, but rapidcheck found a failure (see the added `Empty merge unit` section) which turned out just to be a test bug.  See 314